octoprint_mrbeam/__init__.py

```diff
@@ -163,6 +164,7 @@ def initialize(self):
 		self._ioBeam = ioBeamHandler(self._event_bus, self._settings.get(["dev", "sockets", "iobeam"]))
 		self._temperatureManager = temperatureManager()
 		self._dustManager = dustManager()
+		self.estimation = Estimation(self._event_bus)
```

Comment:
could you please call it JobTimeEsitmation. Then it's really clear what it is about.